tools/collaboration.md

```diff
@@ -0,0 +1,86 @@
+
+## Meetings
+
+Hosted projects and working groups will use the LFX PCC Meeting Management service for all virtual and hybrid project meetings. This tool leverages Zoom, adding helpful features to reduce the overhead for project communities in managing their meetings.
+
+- Ability to automatically invite committee members to a meeting, including sending out invites for existing meetings automatically when a committee member is added and removing the invite when the committee member is removed.
+- Automated meeting recording and transcription, making those available to named meeting participants via their LFX Individual Dashboard and, more broadly, via the Public Meeting Calendar if the meeting is set to `Public`.
+Meeting attendance is taken automatically and attached to the meeting itself. Attendance is reportable to gauge participants' engagement.
+
+### Adding meetings
+
+Instructions for creating a meeting in PCC can be found [here][create meeting instructions]. Specific guidelines for hosted projects and working groups should be as follows.
+
+- Generally, all hosted projects and working group meetings should have `Make Meeting Public` checked and `Restricted` and unchecked, as those meetings are public by default. Occasionally, meetings related to security vulnerability review or other sensitive topics may be held; in those cases, `Make Meeting Public` should be unchecked and `Restricted` and checked.
+- Associate a `Committee` with the meeting to ensure that those committee members are automatically added. Other non-committee members can join using the meeting link and request that a meeting invite be sent to them.
+
+### Cancelling meetings
+
+Instructions for canceling meetings can be found [here][cancel meeting instructions].
+
+## Calendar
+
+Each hosted project and working group has a Public Calendar that can be directly pointed to or embedded in an existing web page. Instructions for getting the link for the hosted project or working group Public Calendar can be found [here][access to public calendar link instructions]. All public meetings for hosted projects and working groups are posted on the [Community Calendar]. 
+
+Meetings that an individual is directly invited to can be found in the 'Meetings' section of that individual's LFX Individual Dashboard; instructions can be found [here][individual dashboard meetings instructions]
+
+## Best Practices
+
+To help individuals not be inundated with calendar overload, try to follow these guidelines:
+
+- Make sure to make meeting invite changes ( including cancellations or reschedulings ) more than 24 hours in advance, as many calendaring clients wait to update. Emailing that calendar's mailing list is also best practice to inform the group of the cancellation.
```

Comment:
>Emailing that calendar's mailing list
Should this be cahnged to "Emailing your project's mailing list" instead?

Reply:
Cleaned this up - let me know if this reads better